js/base/Exchange.js

```diff
@@ -859,6 +859,37 @@ module.exports = class Exchange {
         };
     }
 
+    currencyStructure () {
+        return {
+            'info': undefined,
+            'id': undefined,
+            'lowercaseId': undefined,
+            'uppercaseId': undefined,
```

Comment:
Humm I'm not sure they're 'unified' fields

Reply:
You can ignore those two props. `lowercaseId` is only present in gate, `uppercaseId` only in bitfinex2. those props are just for those exchanges specifically. so, please push the changes (i dont want to alter Sam's PR) and merged this PR when you can.

Reply:
after this PR, please take a look into this another quick dependent PR: https://github.com/ccxt/ccxt/pull/14910/files

Reply:
@samgermain can you remove those two props?

Reply:
> @samgermain can you remove those two props?

ok, I removed it

Reply:
@carlosmiei @samgermain can this PR get merged?